app/src/main/java/mod/jbk/util/OldResourceIdMapper.java

```diff
@@ -2197,7 +2197,7 @@ public static int getDrawableFromOldResourceId(int oldId) {
             case 2131166281:
                 return R.drawable.yellow_arrow_down;
             case 2131166282:
-                return R.drawable.bottom_shadow;
+                return R.drawable.img_nocontent;
```

Comment:
Why did you change it?

Reply:
o arquivo bottom_change fazia parte da lib antiga do FilePicker, como foi tirada ,não tem mais o arquivo

Reply:
> o arquivo bottom_change fazia parte da lib antiga do FilePicker, e já que eu tirei a lib, o arquivo não é mais encontrado